src/WebJobs.Script.WebHost/Standby/StandbyOptions.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost
+{
+    public class StandbyOptions
```

Comment:
Recommend putting a summary block here describing the issue this solves and providing guidance for use, when to prefer this over direct env access, etc.